config.py

```diff
@@ -65,6 +65,11 @@ class Config(object):
     PARTY_PASSWORD = os.getenv('PARTY_PASSWORD')
     PARTY_AUTH = (PARTY_USERNAME, PARTY_PASSWORD)
 
+    REQUESTS_POST_TIMEOUT = os.getenv('REQUESTS_POST_TIMEOUT', 20)
+    NOTIFICATION_TEMPLATE_ID = os.getenv('NOTIFICATION_TEMPLATE_ID', "c69147fc-0f9c-40d4-b111-5b11acc8551f")
```

Comment:
I am working on it , the Template ID has bemoved in the commons. Afterthat will be the real one , possibly just in dev, so that will be able to get tested.